.circleci/integration-tests/Dockerfile.astro_cloud

```diff
@@ -75,7 +75,7 @@ ENV PATH $PATH:$JAVA_HOME/bin::$HIVE_HOME/bin:$HADOOP_HOME/bin
 COPY astronomer-providers /tmp/astronomer-providers
 RUN  python3 -m pip install --upgrade pip
 # Ideally we should install using constraints file
-RUN pip install  --upgrade --force-reinstall  --no-cache-dir /tmp/astronomer-providers[all]
+RUN pip install  --upgrade --no-cache-dir /tmp/astronomer-providers[all]
```

Comment:
The problem with this would be that latest provider code from main not be installed as it would already come installed with the runtime image and that would be non-latest.

Reply:
may I know please what problem are we trying to solve? are we encountering some new issue that we did not as this has been like this since quite a while, no?

Reply:
@pankajkoti correct, and we want these deployment with the main code

Reply:
@pankajkoti more context [here](https://astronomer.slack.com/archives/C01UJJEN0P3/p1699003495317799)

Reply:
okay, are we sure if it's that or the line below causing the reinstall of Airflow?
`RUN pip install apache-airflow[slack]`

How about we try changing this line instead to `RUN pip install apache-airflow-providers-slack` instead of removing the force reinstall?

Reply:
As per CI [logs](https://github.com/astronomer/astronomer-providers/actions/runs/6764683925/job/18383310488#step:4:10685) its getting installed because of reinstall of airflow